lms/envs/test.py

```diff
@@ -448,6 +448,9 @@
         "ENABLE_SHOPPING_CART": True,
         "ENABLE_PAID_COURSE_REGISTRATION": True,
         "SESSION_COOKIE_DOMAIN": "test_microsite.localhost",
+        'ABOUT': 'testmicrosite/about',
```

Comment:
I'm not sure if it's possible given the current implementation, but IMHO it would be better to contain all of these URL overrides in a sub-dictionary, such as:

```
"test_microsite": {
    "urls": {
        "ABOUT": "testmicrosite/about"
    }
}
```